style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
```

Comment:
- [x] Great job adding a .css file :muscle:

Reply:
Thank You